index.html

```diff
@@ -1,230 +1,336 @@
-          <ul>
-            <li>JavaScript</li>
-            <li>Ruby</li>
-            <li>Html</li>
-            <li>CSS</li>
-          </ul>
+        <div class="resume-block">
+          <button type="button" class="resume-text">Get My Resume</button>
         </div>
-      </div>
-      <div class="container">
-        <div class="icon-circle">
-          <img src="Images/icon-frameworks.png" alt="Frameworks-icon" />
+      </section>
+      <div class="about-section">
+        <div class="container">
+          <div class="icon-circle">
+            <img src="Images/icon-languages.png" alt="Language-icon" />
+          </div>
+          <h3 class="about-title">Languages</h3>
+          <div class="card-buttons">
+            <ul>
+              <li>JavaScript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>CSS</li>
+            </ul>
+          </div>
         </div>
-        <h3 class="frameworks">Frameworks</h3>
-        <div class="card-buttons">
-          <ul>
-            <li>Bootstrap</li>
-            <li>Ruby on rails</li>
-            <li>RSpec </li>
-            <li>Capybara</li>
-            <li>Selenium</li>
-          </ul>
+        <div class="container">
+          <div class="icon-circle">
+            <img src="Images/icon-frameworks.png" alt="Frameworks-icon" />
+          </div>
+          <h3 class="frameworks">Frameworks</h3>
+          <div class="card-buttons">
+            <ul>
+              <li>Bootstrap</li>
+              <li>Ruby on rails</li>
+              <li>RSpec</li>
+              <li>Capybara</li>
+              <li>Selenium</li>
+            </ul>
+          </div>
+        </div>
+        <div class="container">
+          <div class="icon-circle">
+            <img src="Images/icon-skills.png" alt="Skills-icon" />
+          </div>
+          <h3 class="about-title">Skills</h3>
+          <div class="card-buttons">
+            <ul>
+              <li>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
+          </div>
         </div>
       </div>
-      <div class="container">
-        <div class="icon-circle">
-          <img src="Images/icon-skills.png" alt="Skills-icon" />
+    </div>
+    <div class="contact-form">
+      <nav class="text-layout">
+          <h4 class="form-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h4>
+      </nav>
+      <form action="https://formspree.io/f/mbjbdrnb" class="form-data">
+        <fieldset>
+          <label><input class="label" type="text" name="first-name" placeholder="Full name" required /></label>
+          <label><input class="label" type="email" name="email" placeholder="Email address" required /></label>
+          <label>
+            <textarea class="bio" rows="3" cols="30" placeholder="Enter text here"></textarea>
```

Comment:
- [ ] kindly add the HTML validation for your `textarea` : 

1. the field should be required 
2. the field should have a max length of 500 characters 

- [ ] don't forget to add the `name` attribute as well to send it to formspree